2018-10-23 00:16:51,326 ERROR: Exception on /auth/login [GET] [in /Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/app/auth/routes.py", line 25, in login
    return render_template('login.html', title='Sign In', form=form)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: login.html
2018-10-23 00:18:19,098 INFO: Microblog startup [in /Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/app/__init__.py:69]
2018-10-23 00:18:21,816 ERROR: Exception on /auth/login [GET] [in /Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/app/auth/routes.py", line 25, in login
    return render_template('login.html', title='Sign In', form=form)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: login.html
2018-10-23 00:19:44,647 INFO: Microblog startup [in /Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/app/__init__.py:69]
2018-10-23 00:19:52,117 ERROR: Exception on /auth/login [GET] [in /Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/app/auth/routes.py", line 25, in login
    return render_template('login.html', title='Sign In', form=form)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: login.html
2018-10-23 00:26:12,481 INFO: Microblog startup [in /Users/falkvandermeirsch/Documents/Python/mega_tutorial/microblog/app/__init__.py:69]
